IM_calculation/Advanced_IM/Models/SAC_Steel_MF_3Story/run.py

```diff
@@ -0,0 +1,155 @@
+"""
```

Comment:
With there being so many run.py-s now we should do some genericisation of these. Maybe James has idea, he's good at generic stuff.